cpp/src/slice2php/Main.cpp

```diff
@@ -190,6 +194,32 @@ CodeVisitor::visitClassDecl(const ClassDeclPtr& p)
 
 bool
 CodeVisitor::visitClassDefStart(const ClassDefPtr& p)
+{
+    string scoped = p->scoped();
+
+    startNamespace(p);
+
+    _out << sp << nl << "if (PHP_VERSION_ID < 70000)";
```

Comment:
PHP up to 8.1 was working fine with the previous code, I would make this change 8.2 only.
```suggestion
    _out << sp << nl << "if (PHP_VERSION_ID < 80200)";
```